### ISSUE TITLE
Camera issue

### ISSUE THREAD

**Issue by Avazbek-Tursunov** [letssolvetogether/OMRChecker#1]
The camera is heating up

**Comment by shreyas29** [comment]
Which smartphone do you use?

**Comment by Avazbek-Tursunov** [comment]
Happened in all Samsung phone

**Comment by shreyas29** [comment]
Yes. The camera is heating a bit in OnePlus phone as well. We will have to check if we can optimize the code.

**Comment by Avazbek-Tursunov** [comment]
Thank you very much for the program. It was a much-needed program. May I donate to the program?

**Comment by shreyas29** [comment]
> Thank you very much for the program. It was a much-needed program. May I donate to the program?

Issues section is not the right place to discuss this. You can contact me on email. Please check support section in README for contact details.
